### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by kiptubei**
In this project, we have built an exclusive clubhouse where your members can write embarrassing posts about non-members. Inside the clubhouse, members can see who the author of a post is but, outside, they can only see the story and wonder who wrote it.

### Models Created and Associations

- Member model
- Post model

- Member model has many posts
- Post model belongs to member
`
Features 
Members can create posts, view all posts and their authors
Non-members cannot create post, can view all posts but not their members
Member and password validations
Posts validation

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+<% @posts.each do |post| %>
+
+<div>
+  <h4><%= post.title.upcase %></h4>
+  <p> <%= post.body %> </p>
+  <% if member_signed_in? %>
+    <p> By: <%= post.member.name %> </p>
+  <% end %>
+</div>
```
Comment (maelfosso) [inline app/views/posts/index.html.erb]: `if..else` it's what we called logic.
It's not allowed in views.
Move that code in **views helpers**

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,6 +10,23 @@
   </head>
 
   <body>
+
+    <% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
+<% end %>
+<% if alert %>
+  <p class="alert alert-danger"><%= alert %></p>
+<% end %>
```
Comment (maelfosso) [inline app/views/layouts/application.html.erb]: Move logic into views helpers

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,6 +10,23 @@
...
+
+<% if member_signed_in? %>
+    <%= link_to "Sign Out", destroy_member_session_path, method: :delete %>
+    <%= link_to "New post", new_post_path %>
+  <% else %>
+    <%= link_to "Sign In", new_member_session_path %> |
+    <%= link_to "Sign Up", new_member_registration_path %>
+<% end %>
```
Comment (maelfosso) [inline app/views/layouts/application.html.erb]: Move that logic into view helpers

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thanks for the job done so far.
The application is working like a charm :smiley_cat: 
However, there are just some small issues. Read the comments left below.

Once the issues solved, kindly submit a new code review request.
Thanks

Mael FOSSO

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (OlawaleJoseph) [reply app/views/layouts/application.html.erb]: Fixed

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team ,  you have done a brilliant job with this project. :clap:
#### You can merge  !!!

**Pull request merged**